x/lockup/client/cli/query.go

```diff
@@ -535,7 +536,48 @@ $ %s query lockup account-locked-longer-duration <address> <duration>
 	return cmd
 }
 
-// GetCmdAccountLockedLongerDurationNotUnlockingOnly returns account locked records with longer duration from unlocking only queue.
+// GetCmdAccountLockedDuration returns account locked records with a specific duration
+func GetCmdAccountLockedDuration() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "account-locked-duration <address> <duration>",
+		Short: "Query account locked records with a specific duration",
+		Long: strings.TrimSpace(
+			fmt.Sprintf(`Query account locked records with a specific duration.
+Example:
+$ %s query lockup account-locked-duration <address> <duration>
+`,
+				version.AppName,
+			),
+		),
+		Args: cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientQueryContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			duration, err := time.ParseDuration(args[1])
+			if err != nil {
+				return err
+			}
+
+			queryClient := types.NewQueryClient(clientCtx)
+
+			res, err := queryClient.AccountLockedDuration(cmd.Context(), &types.AccountLockedDurationRequest{Owner: args[0], Duration: duration})
```

Comment:
Do we have a type constructor for `AccountLockedDurationRequest`?

Reply:
Like a `NewAccountLockedDurationRequest()`? Can you elaborate further on this?

Reply:
Yup, let's use constructors if they exist.

Reply:
I don't believe we currently have a constructor for this type and all other CLI commands seem to mimic this usage of passing the reference of a direct type initialization.